### PR TITLE
LPS-149023: Excludes and includes for ModelPrefilterContributors

### PR DESCRIPTION
**Pull request by BryanEngler**

### CODE REVIEW

**Comment by BryanEngler** [comment]
ci:test:sf

**Comment by BryanEngler** [comment]
ci:test:relevant

**Comment by BryanEngler** [comment]
ci:test:search

**Comment by BryanEngler** [comment]
ci:test:search-remote

**Comment by BryanEngler** [comment]
ci:test:search-remote-minimum

**Comment by BryanEngler** [comment]
ci:forward

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsHolder.java [hunk 1]
```diff
@@ -25,7 +25,7 @@
 public interface ModelPreFilterContributorsHolder {
 
 	public Stream<ModelPreFilterContributor> stream(
-		String entryClassName, Collection<String> excludeIds,
```
Comment (peerkar) [inline modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsHolder.java]: This was fully  intentional and i was aware the other order used there. My understanding is that alphabetical order should be used.

Comment (BryanEngler) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsHolder.java]: ya, i guess i should have sorted all of the _other_ holders =)

**Comment by joshchong** [comment]
ci:test:bundle

**Comment by BryanEngler** [comment]
@peerkar i think we still have the issue of the failing functional test, but does the SF look better now?

**Comment by BryanEngler** [comment]
ci:test:bundle

File: modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsHolder.java (hunk 1 above)
Comment (peerkar) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsHolder.java]: you indeed did it, thanks :) the longer story, the question behind my commit was:  is it consistency over correctness? is it cohesion over convention? Is it better to have all "wrong" over only something "right"?  Intuitively I would have chosen consistency like you did, but was anyways curious to see, how this works out.

Comment (BryanEngler) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsHolder.java]: ya i actually talked to bchan a little bit and really it just depends on the situation. theres no blanket answer that applies to everything. in this case, the sorting/renaming i did was low effort and low risk, so i went for it

Comment (BryanEngler) [reply modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/indexer/ModelPreFilterContributorsHolder.java]: if sorting/renaming the "wrong" old code would have been more effort/risk, then i think the proper approach would have been to make the new code "right", the way you originally had it, even though there was inconsistency with the other classes

**Comment by joshchong** [comment]
@BryanEngler @peerkar, I took a look at the test failure and was able to confirm manually that these changes seem to prevent object types from being returned in search results when a blueprint is applied to the search. This happens both when searching within the blueprints preview and when on the search page with a blueprint applied. This happens with or without an element added to the blueprint. I confirmed that the blueprint I was testing had "All Searchable Types" selected and all Query Contributors were on.

**Pull request closed without merging**